zebra-rpc/src/methods/types/get_block_template.rs

```diff
@@ -522,8 +510,8 @@ where
         }
     }
 
-    /// Returns the miner's address.
-    pub fn miner_address(&self) -> Option<transparent::Address> {
+    /// Returns a valid miner address, if any.
```

Comment:
Nitpick/Optional/Subjective:

```suggestion
    /// Returns a valid miner address, if one was configured.
```

Reply:
I meant this getter to be general, so it doesn't necessarily rely on the address set through the config. The address could be set in some other way.